modules/xfem/src/userobjects/NodeValueAtXFEMInterface.C

```diff
@@ -123,11 +111,18 @@ PointValueAtXFEMInterface::execute()
             ((dynamic_cast<MooseVariable *>(_var))->gradSln())[0];
       }
     }
+    else // When node is outside of computation domain
+    {
+      _values_positive_level_set_side[i] = 0;
+      _values_negative_level_set_side[i] = 0;
+      _grad_values_positive_level_set_side[i] = RealVectorValue(0);
+      _grad_values_negative_level_set_side[i] = RealVectorValue(0);
```

Comment:
Zero might not be a good choice here -- users won't know if the point values are actually zero or the points are not on the interface. I'm not sure what could be a better solution here though.

Reply:
I agree. Maybe `zero` is fine because we never use those values to calculate the velocity.